pyproj/crs.py

```diff
@@ -340,9 +406,24 @@ def get_geod(self):
             in_kwargs["b"] = self.ellipsoid.semi_minor_metre
         return Geod(**in_kwargs)
 
-    def to_proj4_dict(self):
+    @classmethod
+    def from_dict(cls, proj_dict):
```

Comment:
I personally think `to/from_proj4_dict` is more explicit / better.

Reply:
I agree. That's why I originally went with it. But, I changed it to be consistent with `rasterio.CRS`. Better or worse reasoning :man_shrugging: